kml/serdes_gpx.hpp

```diff
@@ -15,6 +15,53 @@ namespace kml
 {
 namespace gpx
 {
+
+class GpxWriter
+{
+public:
+  DECLARE_EXCEPTION(WriteGpxException, RootException);
+
+  class WriterWrapper
+  {
+  public:
+    explicit WriterWrapper(Writer & writer)
+      : m_writer(writer)
+    {}
+    WriterWrapper & operator<<(std::string_view str);
```

Comment:
Isn't operator<< already defined for string_view? Is it (and WriterWrapper) really needed?

If it's not defined, would it be better to define it for Writer class (and remove similar wrappers if necessary in other places)?

Reply:
Yes, operator definition is required, so I moved it to Writer.